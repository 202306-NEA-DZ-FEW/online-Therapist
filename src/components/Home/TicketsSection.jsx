import React from "react";
import TicketCard from "@/components/Home/TicketCard";
import AreYouCouncelor from "./AreYouCouncelor";
const TicketsSection = () => {
    return (
        <div className='container mx-auto p-10'>
            <h2 className='font-normal block text-3xl md:text-4xl rtl:md:text-3xl  rtl:lg:text-4xl uppercase break-words'>
                purchase tickets
            </h2>
            <p className='uppercase text-lg md:text-xl text-black/70 mt-2'>
                purchase tickets that can be used to book appointments!
            </p>
            <div className='grid  sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-7 my-10 '>
                <TicketCard numberOfTickets={5} price={10} />
                <TicketCard numberOfTickets={25} price={40} />
                <TicketCard numberOfTickets={50} price={70} />
            </div>
            <AreYouCouncelor />
        </div>
    );
};

export default TicketsSection;
