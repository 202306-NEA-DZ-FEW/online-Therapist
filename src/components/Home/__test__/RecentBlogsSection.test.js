import renderer from "react-test-renderer";
import TicketsSection from "@/components/Home/TicketsSection";

it("renders correctly", () => {
    const tree = renderer.create(<TicketsSection />).toJSON();
    expect(tree).toMatchSnapshot();
});
