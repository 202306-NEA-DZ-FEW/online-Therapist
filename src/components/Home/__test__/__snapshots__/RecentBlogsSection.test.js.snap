// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container mx-auto p-10"
>
  <h2
    className="font-normal block text-3xl md:text-4xl rtl:md:text-3xl  rtl:lg:text-4xl uppercase break-words"
  >
    purchase tickets
  </h2>
  <p
    className="uppercase text-lg md:text-xl text-black/70 mt-2"
  >
    purchase tickets that can be used to book appointments!
  </p>
  <div
    className="grid  sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-7 my-10 "
  >
    <div
      className="bg-white/95 flex flex-col items-center justify-center text-center py-16 rounded-2xl space-y-6 shadow-2xl md:last-of-type:w-full md:last-of-type:mx-auto md:last-of-type:col-span-2 lg:last-of-type:col-auto lg:last-of-type:w-full"
    >
      <div
        className="flex items-center justify-center space-x-2"
      >
        <span
          className="uppercase text-4xl"
        >
          5
          $ 
          ticket
        </span>
      </div>
      <span
        className="text-base rtl:text-3xl text-black/50"
      >
        10
        $
      </span>
      <div
        className="w-fit"
      >
        <a
          href="/buyTicket"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <button
            className="uppercase cursor-pointer py-2 px-3 text-lg rounded text-black/80 font-light bg-[#2DD3E3]"
          >
            purchase
          </button>
        </a>
      </div>
    </div>
    <div
      className="bg-white/95 flex flex-col items-center justify-center text-center py-16 rounded-2xl space-y-6 shadow-2xl md:last-of-type:w-full md:last-of-type:mx-auto md:last-of-type:col-span-2 lg:last-of-type:col-auto lg:last-of-type:w-full"
    >
      <div
        className="flex items-center justify-center space-x-2"
      >
        <span
          className="uppercase text-4xl"
        >
          25
          $ 
          ticket
        </span>
      </div>
      <span
        className="text-base rtl:text-3xl text-black/50"
      >
        40
        $
      </span>
      <div
        className="w-fit"
      >
        <a
          href="/buyTicket"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <button
            className="uppercase cursor-pointer py-2 px-3 text-lg rounded text-black/80 font-light bg-[#2DD3E3]"
          >
            purchase
          </button>
        </a>
      </div>
    </div>
    <div
      className="bg-white/95 flex flex-col items-center justify-center text-center py-16 rounded-2xl space-y-6 shadow-2xl md:last-of-type:w-full md:last-of-type:mx-auto md:last-of-type:col-span-2 lg:last-of-type:col-auto lg:last-of-type:w-full"
    >
      <div
        className="flex items-center justify-center space-x-2"
      >
        <span
          className="uppercase text-4xl"
        >
          50
          $ 
          ticket
        </span>
      </div>
      <span
        className="text-base rtl:text-3xl text-black/50"
      >
        70
        $
      </span>
      <div
        className="w-fit"
      >
        <a
          href="/buyTicket"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <button
            className="uppercase cursor-pointer py-2 px-3 text-lg rounded text-black/80 font-light bg-[#2DD3E3]"
          >
            purchase
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="font-atkinson mt-20"
  >
    <h2
      className="font-normal block text-3xl md:text-4xl rtl:md:text-3xl  rtl:lg:text-4xl uppercase break-words"
    >
      Join our team
    </h2>
    <p
      className="uppercase text-lg md:text-xl text-black/70 my-2"
    >
      Are you a counselor?
    </p>
    <div
      className="bg-white/95 flex flex-col items-center justify-center text-center py-10 px-5 rounded-md space-y-6 shadow-2xl min-h-[20rem]"
    >
      <span
        className="sm:text-xl text-base text-black/80"
      >
        Interested in becoming a part of our revolutionary mental health platform? We offer you the freedom to set your own schedule and decide how much you want to work. Meanwhile, we take care of all the heavy lifting, from handling client referrals to managing all billing details. Join us and focus on what you do best â€“ providing quality mental health services, while we ensure a seamless experience for both you and your clients.
      </span>
      <div>
        <a
          href="/requirements"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          <button
            className="uppercase cursor-pointer py-2 px-3 text-lg rounded text-black/80 font-light bg-[#2DD3E3]"
          >
            Learn more
          </button>
        </a>
      </div>
    </div>
  </div>
</div>
`;
